### PR TITLE
CIRC-878: Close loans with no fees to charge.

### PR DESCRIPTION
**Pull request by bohdan-suprun**
https://issues.folio.org/browse/CIRC-878 - Backend - Aged to lost: Close loans when no fees charged - SET COST.

## Purpose
Close aged to lost loans when no fee/fines have to be charged immediately. 

## Pre-Merge Checklist:
Before merging this PR, please go through the following list and take appropriate actions.

- Does this PR meet or exceed the expected quality standards?
  - [ ] Code coverage on new code is 80% or greater
  - [ ] Duplications on new code is 3% or less
  - [ ] There are no major code smells or security issues
- Does this introduce breaking changes?
  - [ ] Were any API paths or methods changed, added or removed?
  - [ ] Were there any schema changes?
  - [ ] Did any of the interface versions change?
  - [ ] Were permissions changed, added, or removed?
  - [ ] Are there new interface dependencies?
  - [ ] There are no breaking changes in this PR.
  
If there are breaking changes, please **STOP** and consider the following:

- What other modules will these changes impact?
- Do JIRAs exist to update the impacted modules?
  - [ ] If not, please create them
  - [ ] Do they contain the appropriate level of detail?  Which endpoints/schemas changed, etc.
  - [ ] Do they have all they appropriate links to blocked/related issues?
- Are the JIRAs under active development?  
  - [ ] If not, contact the project's PO and make sure they're aware of the urgency.
- Do PRs exist for these changes?
  - [ ] If so, have they been approved?

Ideally all of the PRs involved in breaking changes would be merged in the same day to avoid breaking the folio-testing environment.  Communication is paramount if that is to be achieved, especially as the number of intermodule and inter-team dependencies increase.  

While it's helpful for reviewers to help identify potential problems, ensuring that it's safe to merge is ultimately the responsibility of the PR assignee.

### CODE REVIEW

File: src/test/java/org/folio/circulation/domain/policy/lostitem/LostItemPolicyTest.java [hunk 1]
```diff
@@ -151,98 +164,6 @@ public void shouldNotAgeItemToLostIfActualCostIsUsed() {
     assertFalse(lostItemPolicy.canAgeLoanToLost(now(UTC)));
   }
 
-  @Test
```
Comment (marcjohnson-kint) [inline src/test/java/org/folio/circulation/domain/policy/lostitem/LostItemPolicyTest.java:154]: Why have all of these tests been removed? Have they been replaced with similar tests elsewhere or are they no longer applicable?

Comment (marcjohnson-kint) [reply src/test/java/org/folio/circulation/domain/policy/lostitem/LostItemPolicyTest.java:154]: Could some unit tests for the `shouldCloseLoan` method be written to replace these, as parts of that logic don't appear to be tested?

Comment (bohdan-suprun) [reply src/test/java/org/folio/circulation/domain/policy/lostitem/LostItemPolicyTest.java:154]: They are not needed anymore, the `shouldChargeFeesWhenAgedToLost` method is removed.

Comment (bohdan-suprun) [reply src/test/java/org/folio/circulation/domain/policy/lostitem/LostItemPolicyTest.java:154]: > Could some unit tests for the shouldCloseLoan method be written to replace these, as parts of that logic don't appear to be tested?

Added.

**Review by marcjohnson-kint — APPROVED** [review]

**Pull request merged**